pyccel/codegen/printing/cwrappercode.py

```diff
@@ -184,41 +189,21 @@ def _get_check_type_statement(self, variable, collect_var):
 
         return check
 
-    # --------------------------------------------------------------------
-    # Functions that take care of creating cast or convert type function call :
-    # --------------------------------------------------------------------
-    def _create_collecting_value_body(self, variable, collect_var, tmp_variable = None):
+    def _get_wrapper_name(self, used_names, func):
         """
-        Create If block to differentiate between python and numpy data types when collecting value
-        format :
-            if (collect_var is numpy_scalar)
-                collect_value from numpy type
-            else
-                collect value from python type
-        Parameters:
-        ----------
-        variable: variable
-            the variable needed to collect
-        collect_var : variable
-            the pyobject variable
-        tmp_variable : variable
-            temporary variable to hold value default None
-
-        Returns
-        -------
-        body : If block
+        create wrapper function name
```

Comment:
Could you add `Parameters` to this docstring please

Reply:
done 637c2bd